backend/geonature/core/gn_meta/routes.py

```diff
@@ -338,22 +331,21 @@ def sensi_report():
         .outerjoin(
             TNomenclatures, TNomenclatures.id_nomenclature == Synthese.id_nomenclature_sensitivity
         )
-        .filter(LAreas.id_type == func.ref_geo.get_id_area_type("DEP"))
+        .where(LAreas.id_type == func.ref_geo.get_id_area_type("DEP"))
+        .where(Synthese.id_module == id_module if id_module else True)
```

Comment:
```suggestion
        .where(Synthese.id_module == id_module if id_module is not None else True)
```
ou idem avec `where_if`